bindings/metadata_shortener/rust/src/lib.rs

```diff
@@ -44,6 +51,11 @@ fn Java_io_novafoundation_nova_metadata_1shortener_MetadataShortener_generate_1e
     signed_extras: jbyteArray,
     additional_signed: jbyteArray,
     metadata: jbyteArray,
+    spec_version: jint,
+    spec_name: JString,
+    base58_prefix: jint,
+    decimals: jint,
+    token_symbol: JString,
```

Comment:
I updated extrinsic proof generation to include additional information to match zondax implementation: https://github.com/Zondax/ledger-polkadot-generic-api/blob/master/rust/src/lib.rs#L177